modules/nw-ingress-sharding-route-configuration.adoc

```diff
@@ -0,0 +1,111 @@
+// Module included in the following assemblies:
+//
+// * configuring_ingress_cluster_traffic/configuring-in4ress-cluster-traffic-ingress-controller.adoc
```

Comment:
Typo: in4ress

Reply:
One of my weirder typos...thanks for catching that!